test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -31,27 +32,58 @@ using namespace shared_model::crypto;
 using namespace std::literals::string_literals;
 
 namespace integration_framework {
+
+  const std::string IntegrationTestFramework::kDefaultDomain = "test";
```

Comment:
Why it starts with `k`?

Reply:
[Due to code style](https://google.github.io/styleguide/cppguide.html#Constant_Names)